### PR TITLE
Update language files for improved localization

### PR DESCRIPTION
**Pull request by miladsoft**
- Updated translations for all supported languages
- Ensured consistency and accuracy across language files
- Added missing translations and corrected existing ones
- Optimized the structure for better maintainability

This update enhances the overall user experience by providing more accurate and comprehensive localization.

### CODE REVIEW

File: angular/src/assets/i18n/fr.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "en": "English",
-  "no": "Norsk",
+  "no": "Norwegian",
```
Comment (sondreb) [inline angular/src/assets/i18n/fr.json:3]: This should be "Norsk". These are local names, if you are Russian or Norwegian, you want to see your local language here.

**Review by sondreb — CHANGES_REQUESTED** [review]
Thanks, but see comment.

File: angular/src/assets/i18n/fr.json (hunk 1 above)
Comment (miladsoft) [reply angular/src/assets/i18n/fr.json:3]: image

**Review by sondreb — APPROVED** [review]
Thank you for this effort!

**Pull request merged**